summit/app/index.html

```diff
@@ -65,8 +65,8 @@
 
             <div class="flex horizontal center-center layout">
               <iron-selector class="wide-only" selected="[[page]]" activate-event="click" attr-for-selected="name">
-                <a name="schedule" class="tab" href="/summit/schedule">SCHEDULE</a>
-                <a name="speakers" class="tab" href="/summit/speakers">SPEAKERS</a>
+                <a name="schedule" on-tap="_pauseLivestream" class="tab" href="/summit/schedule">SCHEDULE</a>
```

Comment:
Are there any other ways besides `tap`-ing on those links that someone might navigate off the page hosting the livestream that would also necessitate calling `_pauseLivestream()`?

Maybe you can listen for a change to `page` and use that as a signal to pause?